packages/aws-cdk-lib/aws-apigatewayv2-integrations/README.md

```diff
@@ -303,4 +303,7 @@ webSocketApi.addRoute('$connect', {
     },
   }),
 });
-```
\ No newline at end of file
+```
+
+You can also set additional properties to change the behavior of your integration, such as `contentHandling`.
+See [Working with binary media types for WebSocket APIs](https://docs.aws.amazon.com/apigateway/latest/developerguide/websocket-api-develop-binary-media-types.html).
```

Comment:
worth adding `timeout` here too?

Reply:
There are already a few existing optional props on top of the two I've added:
https://github.com/nmussy/aws-cdk/blob/58625787bd0027bb375a760271c5fed957ac6966/packages/%40aws-cdk-testing/framework-integ/test/aws-apigatewayv2-integrations/test/websocket/integ.aws.ts#L44-L61

`timeout` feels pretty minor, especially because its default is already maxed out. Most of them are already shown in the "AWS WebSocket Integration" example, I could add the `timeout` there?

https://github.com/nmussy/aws-cdk/blob/58625787bd0027bb375a760271c5fed957ac6966/packages/aws-cdk-lib/aws-apigatewayv2-integrations/README.md?plain=1#L289-L305